commit 46ba8d1156e78b4d05c3f05d8d902832bfb33965
Author: Wei Yongjun <weiyongjun1@huawei.com>
Date:   Sat May 26 09:45:59 2018 +0000

    misc: ibmvmc: Use GFP_ATOMIC under spin lock
    
    [ Upstream commit 97b715b62e5b4c6edb75d023f556fd09a46cb4e1 ]
    
    The function alloc_dma_buffer() is called from ibmvmc_add_buffer(),
    in which a spin lock be held here, so we should use GFP_ATOMIC when
    a lock is held.
    
    Fixes: 0eca353e7ae7 ("misc: IBM Virtual Management Channel Driver (VMC)")
    Signed-off-by: Wei Yongjun <weiyongjun1@huawei.com>
    Reviewed-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/misc/ibmvmc.c b/drivers/misc/ibmvmc.c
index 50d82c3d032a..b8aaa684c397 100644
--- a/drivers/misc/ibmvmc.c
+++ b/drivers/misc/ibmvmc.c
@@ -273,7 +273,7 @@ static void *alloc_dma_buffer(struct vio_dev *vdev, size_t size,
 			      dma_addr_t *dma_handle)
 {
 	/* allocate memory */
-	void *buffer = kzalloc(size, GFP_KERNEL);
+	void *buffer = kzalloc(size, GFP_ATOMIC);
 
 	if (!buffer) {
 		*dma_handle = 0;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1092_46ba8d1156e78b4d05c3f05d8d902832bfb33965.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1092_46ba8d1156e78b4d05c3f05d8d902832bfb33965.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
